web/src/OverviewResourcePane.tsx

```diff
@@ -32,7 +32,7 @@ let Main = styled.div`
 export default function OverviewResourcePane(props: OverviewResourcePaneProps) {
   let nav = useTabNav()
   let resources = props.view?.resources || []
-  let name = nav.candidateTab || nav.selectedTab || ""
+  let name = nav.invalidTab || nav.selectedTab || ""
```

Comment:
Checking my understanding: So, it's possible that you have an invalid tab (e.g., resource no longer exists). We figure this out by checking the name provided by the URL against a list of current resources. Then, if it's invalid, we do stuff like show "resource not found" or whatnot.

Reply:
yup! exactly